### PR TITLE
Create Karttikeya

### PR DESCRIPTION
**Pull request by KartKernel**
## Requirements
Unless explicitly specified otherwise by a **maintainer** or in the requirement description, your domain must pass **ALL** the indicated requirements above.

Please note that we reserve the rights not to accept any domain at our own discretion.

- [x] The file is in the `domains` folder and is in the JSON format.
- [x] The file's name is all lowercased and alphanumeric. 
- [x] You have completed your website. 
- [x] The website is reachable.  
- [x] You're not using Vercel.  
- [ ] The CNAME record doesn't contain `https://` or `/`.  
- [ ] There is sufficient information at the `owner` field.

### CODE REVIEW

File: domains/Karttikeya
Comment (DEV-DIBSTER) [inline domains/Karttikeya:1]: File doesn't end in .json.

File: domains/Karttikeya [hunk 1]
```diff
@@ -0,0 +1,11 @@
+{
+  "description": "Karttikeya's personal developer website",
+  "repo": "https://github.com/KartKernel/KartKernel.github.io",
+  "owner": {
+    "username": "Karttikeya",
+    "email": "karttikeyashahi@gmail.com"
+  },
+  "record": {
+    "CNAME": "KartKernel.github.io"
```
Comment (DEV-DIBSTER) [inline domains/Karttikeya:9]: Cannot seem to access your site.

**Review by DEV-DIBSTER — DISMISSED** [review]

File: domains/Karttikeya
Comment (orxngc) [reply domains/Karttikeya:1]: user changed file name

File: domains/Karttikeya (hunk 1 above)
Comment (orxngc) [reply domains/Karttikeya]: site is now accessible

**Comment by KartKernel** [comment]
> The filename must be all lowercase. That is, `karttikeya.json` instead of `Karttikeya.json`. Change that and then I can merge it, thank you!

Hi, I have changed the name, thank you

**Pull request merged**